set.mm

```diff
@@ -231100,6 +231343,14 @@ an extension of the previous (inserting an element and its inverse at
   ablgrp $p |- ( G e. Abel -> G e. Grp ) $=
     ( cabl wcel cgrp ccmn isabl simplbi ) ABCADCAECAFG $.
 
+  ${
+    ablgrpd.1 $e |- ( ph -> G e. Abel ) $.
+    $( Deduction associated with ~ ablgrp .  (Contributed by Rohan Ridenour,
+       3-Aug-2023.) $)
```

Comment:
Comment: "An Abelian group is a group, deduction form"

Reply:
Done. The better way would be "abelian group" (with lowercase), but set.mm seems to use both "Abelian" and "abelian" at different places.